It looks like this breaks the mingw builds.  The clang 64-bit builds fail as shown below but this is something I can quickly fix.  The typecast here is safe.  I can patch that here by adding a (GB_void *) typecast to the S_input parameter.  I'll need to make the related change in my dev2 branch of GraphBLAS, however, since the SuiteSparse copy of GraphBLAS is the latest stable one, not my development branch for GraphBLAS.

I'm not sure why the mingw32 should fail; it was working before but it didn't fine the openblas64 library somehow.

```
CMakeFiles/GraphBLAS.dir/Source/extract/GB_I_inverse.c.obj -c C:/a/SuiteSparse/SuiteSparse/GraphBLAS/Source/extract/GB_I_inverse.c
  C:/a/SuiteSparse/SuiteSparse/GraphBLAS/Source/extract/GB_I_inverse.c:123:9: error: incompatible pointer types passing 'uint64_t[1]' (aka 'unsigned long long[1]') to parameter of type 'const GB_void *' (aka 'const unsigned char *') [-Wincompatible-pointer-types]
    123 |         S_input,            // values of R (iso-valued)
        |         ^~~~~~~
  C:/a/SuiteSparse/SuiteSparse/GraphBLAS/Source/builder/GB_build.h:50:29: note: passing argument to parameter 'S_input' here
     50 |     const GB_void *restrict S_input,// array of values of tuples, size nvals,
        |                             ^
  1 error generated.
```

--- a/GraphBLAS/Source/extract/GB_I_inverse.c
+++ b/GraphBLAS/Source/extract/GB_I_inverse.c
@@ -120,7 +120,7 @@ GrB_Info GB_I_inverse           // invert the I list for C=A(I,:)
         // J_input
         I,                  // row indices are in I [0..nI-1]
         // S_input
-        S_input,            // values of R (iso-valued)
+        (const GB_void *) S_input,  // values of R (iso-valued)
         // S_iso
         true,               // R is iso-valued
         // nvals

--- a/GraphBLAS/Source/get_set/GrB_Type_set.c
+++ b/GraphBLAS/Source/get_set/GrB_Type_set.c
@@ -99,7 +99,7 @@ GrB_Info GrB_Type_set_VOID
     if (field == GxB_PRINT_FUNCTION && type->code == GB_UDT_code &&
         size == sizeof (GxB_print_function))
     { 
-        type->print_function = (GxB_print_function *) value ;
+        type->print_function = (GxB_print_function) value ;
         return (GrB_SUCCESS) ;
     }
 

